src/components/order-service/order-service.controller.ts

```diff
@@ -312,6 +313,34 @@ export class OrderServiceController implements angular.IController {
     this.ctrl.parameterSchema = schema;
   }
 
+  private updateParameterForm() {
+    let formDefinition: any = _.get(this, 'ctrl.selectedPlan.externalMetadata.schemas.service_instance.create.form_definition');
```

Comment:
Let's add a prefix to avoid ever colliding with a future field added to OSB API. I'd suggest `openshift_form_definition`. That also makes it clear to anyone looking at the broker that the field is specific to the OpenShift catalog.

@pmorie Any objections?